### PR TITLE
Use catalog JSON-LD instead of Bibdata.

### PR DESCRIPTION
**Pull request by tpendragon**
Work towards #5229

### CODE REVIEW

File: app/models/processed_event.rb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
-# This resource represents a single bibdata dump and tracks whether it has been
+# This resource represents a single catalog dump and tracks whether it has been
 #   processed to refresh metadata from voyager
-# We need to do this because bibdata provides an ongoing list of dump events,
+# We need to do this because catalog provides an ongoing list of dump events,
```
Comment (hackartisan) [inline app/models/processed_event.rb:4]: Wait, catalog doesn't have dumps

**Review by hackartisan — CHANGES_REQUESTED** [review]
you re-pulled all these fixtures?

**Comment by hackartisan** [comment]
this is pretty amazing that it just drops in like that!

**Comment by tpendragon** [comment]
Nope, didn't repull them, find and replace just got ahead of itself. I'll patch it later, thanks.

**Review by hackartisan — CHANGES_REQUESTED** [review]
This was clearly a big job, thank you for doing it.

It seems like the change of title value may be pretty noticeable to users. It looks like maybe we used to pull just the `245a` (or maybe `a` and `b`) and now we have the whole `245` including medium (`245h`) and statement of responsibility (`245c`). Given that this will then percolate out to, e.g. DPUL, I'm a little concerned this isn't what we want. How can we get assurances that this will meet user expectations? Or do we need to request an update to the json from the catalog?

**Comment by tpendragon** [comment]
@hackartisan You're talking about for titles? This changed a long time ago - we just never updated our fixtures. See for instance https://bibdata.princeton.edu/bibliographic/991234563506421/jsonld

**Comment by hackartisan** [comment]
Okay, great!

**Review by hackartisan — APPROVED** [review]

**Pull request merged**